### PR TITLE
Transform item's path to lowercase in navigator

### PR DESCRIPTION
**Pull request by marinaaisa**
Bug/issue #122911237, if applicable: 

## Summary

Transform item's path to lowercase in navigator, so if symbols have a path with capital letters, the highlighting of the active element in the navigator continues working

## Dependencies

NA

## Testing

Steps:
1. Run a doccarchive with a navigator
2. Make sure that everything works as expected

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [x] Added tests
- [x] Ran `npm test`, and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

File: src/components/Navigator/NavigatorCardItem.vue [hunk 1]
```diff
@@ -89,7 +89,7 @@
         :is="refComponent"
         :id="item.uid"
         :class="{ bolded: isBold }"
-        :url="isGroupMarker ? null : (item.path || '')"
+        :url="isGroupMarker ? null : (item.path.toLowerCase() || '')"
```
[the remark this answers is not in the document]
Comment (marinaaisa) [reply src/components/Navigator/NavigatorCardItem.vue]: Yes, it sounds like a better approach. I changed it https://github.com/apple/swift-docc-render/pull/836/commits/82c3a22f1aebc689804e5efc865c8f44cc2a43f7

It's a pity that https://github.com/vuejs/vue-router/pull/3657 didn't get merged, it made sense to me.

**Comment by marinaaisa** [comment]
@swift-ci test

File: src/components/Navigator/NavigatorCard.vue [hunk 2]
```diff
@@ -247,6 +247,8 @@ export default {
     };
   },
   computed: {
+    isSameRoute: ({ technologyPath, $route }) => (
```
Comment (mportiz08) [inline src/components/Navigator/NavigatorCard.vue:250]: The name is a little ambiguous as to what the comparison is for "same".

```suggestion
    isTechnologyRoute: ({ technologyPath, $route }) => (
```

**Review by mportiz08 — APPROVED** [review]

File: src/components/Navigator/NavigatorCard.vue (hunk 2 above)
Comment (marinaaisa) [reply src/components/Navigator/NavigatorCard.vue]: fixed!

**Comment by marinaaisa** [comment]
@swift-ci test

**Pull request merged**